playlist.c

```diff
@@ -43,8 +43,12 @@
 #endif
 
 #ifndef PLAYLIST_ENTRIES
+#ifdef HAVE_ENTRYSTATES
+#define PLAYLIST_ENTRIES 7
```

Comment:
I did not explicitly disable entry states for old-style playlist. I just don't read or write the entry_slot so that it defaults to 0 which means disabled.